src/components/ListTemplate/ColumnTemplates/DefaultColumn.js

```diff
@@ -12,13 +16,17 @@ export const DefaultColumn = memo(({
   className = sharedCss.cellAppIcon,
   iconKey,
```

Comment:
Lets replace `iconKey` prop with `entityKey` and then on the 45 line pass `entityKey` to the `<AppIcon>` component's `iconKey` prop. In this case, you don't need [this manipulation](https://github.com/folio-org/ui-data-import/pull/535/files#diff-c7438818fbd56fd424a8c77cebbe3a9bR28) and this help to avoid problems, because `iconKey` may differ from the entity name, as a result of which we can get an invalid link.
```suggestion
  entityKey,
```

Reply:
new object in constants.js has been created for profile type mapping